hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1701,6 +1704,18 @@ message DeleteSnapshotRequest {
   optional uint64 deletionTime = 4;
 }
 
+message SnapshotMoveDeletedKeysRequest {
+  optional SnapshotInfo fromSnapshot = 1;
+  optional SnapshotInfo nextSnapshot = 2;
+  repeated SnapshotMoveKeyInfos activeDBKeys = 3;
+  repeated SnapshotMoveKeyInfos nextDBKeys = 4;
```

Comment:
nit:  Naming could be consistent for readability.  you just need "FromSnapshot" DBKeys. Why do you need nextDBKeys ? why do we need activeDBKeys ?

Reply:
`activeDBKeys` are keys that should be moved to active db's `deletedTable`. `nextDBKeys` are keys that should be moved to nextSnapshot's `deletedTable`